features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/LifeCycle/CheckboxLabels.jsx

```diff
@@ -138,7 +143,7 @@ export default function CheckboxLabels(props) {
                         </Grid>
                     </Grid>
                     <Grid xs={12} className={classes.labelsGrid}>
-                        <Typography variant='h10' >
+                        <Typography variant='h6' style={{ fontWeight: 600 }}>
```

Comment:
shall we take this value from theme object?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/7535/commits/419a0f5140ea1caa34abf2805f4f28937054cfd5